spyder/plugins/mainmenu/api.py

```diff
@@ -105,8 +105,9 @@ class HelpMenuSections:
 
 class ApplicationMenu(SpyderMenu):
     """
-    Spyder Main Window application Menu.
+    Spyder main window application menu.
 
-    This class provides application menus with some predefined functionality
-    and section definition.
+    This class provides application menus with some predefined functionality.
     """
+
+    APP_MENU = True
```

Comment:
This is necessary now to show/hide icons on Mac (see the `SpyderMenu._set_icons` method)